tslint.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": [
+    "tslint:latest",
+    "tslint:recommended",
+    "tslint-react",
+    "tslint-eslint-rules"
+  ],
+  "rulesDirectory": [],
+  "rules": {
+    "quotemark": [
+      true,
+      "single"
+    ],
+    "jsx-no-lambda": false,
+    "no-submodule-imports": false,
+    "jsx-no-multiline-js": false,
+    "jsx-boolean-value": false,
+    "jsx-wrap-multiline": false,
+    "object-literal-sort-keys": false
```

Comment:
I also find these rules useful:

Requiring interface names to start with a capital 'I' doesn't add any semantic meaning to the code:
```
"interface-name": [
  true,
  "never-prefix"
],
```

GraphQL template strings often include trailing whitespace:
```
"no-trailing-whitespace": [
  true,
  "ignore-template-strings"
],
```

Reply:
I also like to set `max-line-length` to 140 or so, but that's strictly a matter of personal taste. I can see keeping it at the default for this package.

Reply:
I'll make the 140 line change, I like it a big wider too. 

I think it's pretty useful to require interface names start with `I` instead of postfixing with `Interface`. Less characters to type and differentiates interfaces from other names in the code.

Reply:
I don't prefix interface names with `I` or postfix them with `Interface`, because I don't prefix type names with `T` or postfix them with `Type`. If practical, I think all "names" should be treated equally - the fact of whether a name refers to an interface or a type should not be a significant factor in understanding the meaning of code.

Reply:
I understand. What would name an interface for a react component. `MyComponentNameProps`?

Reply:
Yes, for example if I have a React component named `MyComponent`, then I would probably name its props interface `MyComponentProps`. I think that requiring the name to be `IMyComponentProps` would be a mistake.

Reply:
Alright. Will make changes, update package and merge.